wgpu-hal/src/lib.rs

```diff
@@ -58,7 +58,7 @@ mod dx11;
 mod dx12;
 mod empty;
 #[cfg(all(feature = "gles"))]
-mod gles;
+pub mod gles;
```

Comment:
This should probably be an all or nothing deal - if we want to make the backends public, they should all be public.

Reply:
Cool, done in [9bd69ee](https://github.com/gfx-rs/wgpu/pull/2609/commits/9bd69ee5de8223cf54be719d0a37af11ad763154).